app/initializers/export-application-global.js

```diff
@@ -1,7 +1,8 @@
 import Ember from 'ember';
 import config from '../config/environment';
 
-export function initialize(container, application) {
+export function initialize() {
+  var application = arguments[1] || arguments[0];
```

Comment:
@rwjblue @stefanpenner if this is an acceptable fix, I'm happy to apply this pattern to ember's internal initializers. Canary is currently throwing warnings, so I assume they'll need to be handled before cutting the next beta